src/widgets/Markdown/Markdown.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { TextArea } from 'semantic-ui-react';
+import ReactMarkdown from 'react-markdown';
+
+import './markdown.scss';
+
+class Markdown extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: props.value
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event, { value }) {
+    this.setState({ value });
```

Comment:
Correct me if i'm wrong but the state in this component seems useless. Why can't you pass props.value and when the `TextArea` updates you will call props.onChange? If it is unnecessary I would create a stateless component because otherwise the props and the state can (in theory) get out of sync.

Reply:
i will look into this